src/constants.ts

```diff
@@ -44,6 +44,7 @@ export const LEGACY_CHE_NAMESPACE = 'che'
 // OLM
 export const DEFAULT_CHE_OLM_PACKAGE_NAME = 'eclipse-che'
 export const OLM_STABLE_CHANNEL_NAME = 'stable'
+export const OLM_STABLE_ALL_NAMESPACES_CHANNEL_NAME = 'tech-preview-stable-all-namespaces'
```

Comment:
What about `tech-preview-all-namespaces` channel ?

Reply:
To me having `tech-preview` and `stable` together is controversy.
I left `stable` in the name because this channel was originally supposed to be stable and we have plans to add something like `next-all-namespaces` soon.